components/submissions/SubmissionsTable.tsx

```diff
@@ -0,0 +1,90 @@
+import { StatusBadge } from "./StatusBadge";
+
+;
+
+export const SubmissionsTable = () => {
+    const rows = [
+        {
+            title: 'Hepatic Infraction',
+            doctor: '',
+            createdAt: '22/02/2021',
+            status: 'pending',
+        },
+        {
+            title: 'Pancreatis Acute',
+            doctor: 'Dr. John Doe',
+            createdAt: '22/02/2021',
+            status: 'in_progress',
+        },
+        {
+            title: 'Bone marrow failure',
+            doctor: 'Dr. John Doe',
+            createdAt: '22/02/2021',
+            status: 'done',
+        },
+    ];
+
+    return (
+        <>
+            <div className="flex justify-end mt-6">
+                <select 
+                    id="statusSelectBox" 
+                    className="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500 w-1/5 mb-2">
```

Comment:
I don't personally like having a line that stretches for this much. This is just personal opinion but I prefer using the string template comma and just add a couple of line breaks to make this more readable